ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Release/ManualInterventionSample.cs

```diff
@@ -0,0 +1,187 @@
+                        {
+                            DaysToKeep = 30,
+                            ReleasesToKeep = 3,
+                            RetainBuild = true
+                        }
+                    }
+                }
+            };
+
+            // Get a release client instance
+            VssConnection connection = Context.Connection;
+            ReleaseHttpClient releaseClient = connection.GetClient<ReleaseHttpClient>();
+
+            // create a release definition
+            ReleaseDefinition releaseDefinition = releaseClient.CreateReleaseDefinitionAsync(project: projectName, releaseDefinition: definition).Result;
+            this._newlyCreatedReleaseDefinitionId = releaseDefinition.Id;
+
+            this._newlyCreatedRelease = ReleasesSample.CreateRelease(releaseClient, _newlyCreatedReleaseDefinitionId, projectName);
+
+            Console.WriteLine("{0} {1}", _newlyCreatedRelease.Id.ToString().PadLeft(6), _newlyCreatedRelease.Name);
+
+            return _newlyCreatedRelease;
+        }
+
+        [ClientSampleMethod]
+        public IList<ManualIntervention> GetManualInterventions()
+        {
+            Debugger.Launch();
+            string projectName = ClientSampleHelpers.FindAnyProject(this.Context).Name;
+
+            // Get a release client instance
+            VssConnection connection = Context.Connection;
+            ReleaseHttpClient releaseClient = connection.GetClient<ReleaseHttpClient>();
+
+            // Get all manual interventions
+            IList<ManualIntervention> manualInterventions = releaseClient.GetManualInterventionsAsync(project: projectName, releaseId: this._newlyCreatedRelease.Id).Result;
+            foreach (ManualIntervention manualIntervention in manualInterventions)
+            {
+                Console.WriteLine("{0} {1}", manualIntervention.Id.ToString().PadLeft(6), manualIntervention.Name);
+            }
+
+            this._manualInterventions = manualInterventions;
+
+            return manualInterventions;
+        }
+
+        [ClientSampleMethod]
+        public ManualIntervention GetManualIntervention()
+        {
+            string projectName = ClientSampleHelpers.FindAnyProject(this.Context).Name;
+
+            // Get a release client instance
+            VssConnection connection = Context.Connection;
+            ReleaseHttpClient releaseClient = connection.GetClient<ReleaseHttpClient>();
+
+            // Get a manual intervention
+            ManualIntervention manualIntervention = releaseClient.GetManualInterventionAsync(project: projectName, releaseId: this._newlyCreatedRelease.Id, manualInterventionId: this._manualInterventions.FirstOrDefault().Id).Result;
+            Console.WriteLine("{0} {1}", manualIntervention.Id.ToString().PadLeft(6), manualIntervention.Name);
+
+            return manualIntervention;
+        }
+
+        [ClientSampleMethod]
+        public ManualIntervention UpdateManualIntervention()
```

Comment:
Have a sample for resume and sample for reject?

Reply:
done